packages/async-rewriter/src/async-writer-babel.ts

```diff
@@ -106,8 +106,23 @@ var TypeInferenceVisitor: Visitor = { /* eslint no-var:0 */
             throw new MongoshInvalidInputError(`Cannot access Mongosh API types dynamically. ${help}`);
           }
           path.node['shellType'] = state.symbols.signatures.unknown;
+          debug(`MemberExpression: { object.sType: ${lhsType.type}, property.name: ${rhs} }`, path.node['shellType']);
           return;
       }
+      if (path.node.object.type === 'ThisExpression') {
+        const classPath = path.findParent((p) => p.isClassDeclaration());
+        if (!classPath) {
+          throw new MongoshUnimplementedError('Unable to handle this outside of method definition of class declaration');
```

Comment:
I'd put "this" in quotation marks. I had to read the sentence a couple of times to understand what you mean.

Reply:
Done!